api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/base_resource_controller.rb

```diff
@@ -13,6 +13,11 @@ module MnoEnterprise::Concerns::Controllers::Jpi::V1::Admin::BaseResourceControl
 
   protected
 
+  def act_as_manager
+    account_manager_enabled = MnoEnterprise::Tenant.show&.frontend_config.dig('admin_panel', 'account_manager', 'enabled')
```

Comment:
Use `Settings&.admin_panel&.account_manager&.enabled`

FYI, I merged a PR that details a bit more the feature flags: https://github.com/maestrano/mno-enterprise/blob/4.0/docs/feature-flags.md